SpatialGDK/Source/SpatialGDK/Public/Utils/SchemaDatabase.h

```diff
@@ -27,6 +27,9 @@ struct FSchemaData
 {
 	GENERATED_USTRUCT_BODY()
 
+	UPROPERTY(VisibleAnywhere)
+	FString GeneratedSchemaName;
```

Comment:
Can this be EDITOR_ONLY_DATA?

Reply:
Potentially. Not sure how non-editor commandlets behave with that, and shouldn't be too much of a burden yet.